conda.recipe/saspy/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{%set name = "saspy" %}
```

Comment:
I committed the recipes for now in case you wanted to test them. After the other changes on this PR are ok I was planning on removing the recipes from the PR and commit them to [conda-forge](https://github.com/conda-forge/) where me and other can maintain them.